database/index.js

```diff
@@ -20,14 +25,19 @@ let save = (arr) => {
     let entry = arr[i];
     let toSave = new Review({
       listingId: entry.listingId,
+      location: entry.location,
       reviewId: entry.reviewId,
       date: entry.date,
+      latest: entry.latest,
+      howRecent: entry.howRecent,
       rating: entry.rating,
       reviewHeadline: entry.reviewHeadline,
       comment: entry.comment,
       userName: entry.userName,
       userLocation: entry.userLocation,
-      userImage: entry.userImage
+      userImage: entry.userImage,
+      userThumbs: entry.userThumbs,
+      userReviews: entry.userReviews
     });
```

Comment:
This seems like it may be unnecessary doubling. Does the entry already match the Review schema format? If so you can just say toSave = new Review(entry). Is there a reason you don't do that?

Reply:
will make this change and update